definitions/checks/foreman/puppet_class_duplicates.rb

```diff
@@ -0,0 +1,49 @@
+module Checks
+  module Foreman
+    class PuppetClassDuplicates < ForemanMaintain::Check
+      metadata do
+        label :puppet_class_duplicates
+        for_feature :foreman_database
+        description 'Check for duplicate Puppet class records'
+        tags :pre_upgrade
+        confine do
+          feature(:foreman_database) &&
+            feature(:foreman_database).check_max_version('foreman', '1.20')
```

Comment:
actually, this confine block could be eliminated to:

```
confine do
  check_max_version('foreman', '1.20')
end
```

Reply:
The `for_feature` already implies that the feature is there, and the `check_max_version` should be available in this context already